docs/ops/arithmetic/Add_1.md

```diff
@@ -6,46 +6,48 @@
 
 **Short description**: *Add* performs element-wise addition operation with two given tensors applying multi-directional broadcast rules.
 
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Add* performs addition operation for the input tensors *a* and *b* using the formula below:
+
+\f[
+o_{i} = a_{i} + b_{i}
+\f]
+
 **Attributes**:
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
-    * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>.
+    * *none* - no auto-broadcasting is allowed, all input shapes must match,
+    * *numpy* - numpy broadcasting rules, description is available in [broadcast_rules](../broadcast_rules.md).
   * **Type**: string
   * **Default value**: "numpy"
   * **Required**: *no*
 
 **Inputs**
 
-* **1**: A tensor of type T. Required.
-* **2**: A tensor of type T. Required.
+* **1**: A tensor of type T and arbitrary shape and rank. **Required.**
+* **2**: A tensor of type T and arbitrary shape and rank. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise addition operation. A tensor of type T.
+* **1**: The result of element-wise addition operation. A tensor of type T with shape equal to broadcasted shape of the two inputs. When both inputs are of rank = 0 result is a scalar.
```

Comment:
Regarding "When both inputs are of rank = 0 result is a scalar.". Let's describe this case in the broadcast_rules.md and do not mention here.

Reply:
Rule added to `broadcast_rules.md`.